src/dependency.rs

```diff
@@ -1,79 +1,7 @@
-use crate::config::Config;
```

Comment:
Good that you are dropping the lib.rs :+1:

Reply:
I hope it will be less confusing